pylib/cqlshlib/cql3handling.py

```diff
@@ -1501,6 +1505,32 @@ def alter_type_field_completer(ctxt, cass):
                        ;
 '''
 
+syntax_rules += r'''
+<slaName> ::= <identifier>
+             | <quotedName>
+             | <unreservedKeyword>
+             ;
+
+<createSlaStatement> ::= "CREATE" "SERVICE_LEVEL" <slaName>
```

Comment:
create statement has "if not exists" option:
https://enterprise.docs.scylladb.com/stable/using-scylla/workload-prioritization.html#:~:text=%27primary%27).-,Syntax,-CREATE%20SERVICE_LEVEL%20%5BIF

Reply:
added the optional if not exists